fr.txt

```diff
@@ -0,0 +1,405 @@
+VerificationEmailFailureTitle = "Vérification incorrecte"
+VerificationEmailFailureDescription = "Oups, il semble que le lien qui vous a amené ici n'est pas valide."
+
+; Signup Complete
+; https://accounts.klei.com/signup/complete
+
+RegistrationComplete = "Vérification réussie"
+RegistrationCompleteDescription = "Merci d'avoir validé votre courriel.</p><p>C'est tout bon! Vous pouvez maintenant retourner à votre jeu et jouer en ligne. Si vous avec des suggestions, retours ou besoin de support pendant la bêta, rendez-vous sur le Forum de la Communauté Klei."
+VisiCommunityForum = "Se rendre sur le Forum de la Communauté Klei"
+
+; Login Page
+; https://accounts.klei.com/login
+
+Login = "Se connecter"
+Logout = "Se déconnecter"
+LoginTitle = "Connexion au compte Klei"
+LoginDescription = "Connectez-vous à votre compte Klei pour voir tous vos jeux Klei, échanger objets avec des amis, s'enregistrer pour les bêta et avoir accès à du contenu exclusif.</p><p>Choisissez une plateforme de connexion:"
+
+; User Info Page
+; https://accounts.klei.com/account/info
+
+UserInfo = "Info joueur"
+Preferences = "Préférences"
+UsefulLinks = "Liens utiles"
+KleiUserID = "ID Utilisateur Klei"
+FirstSeen = "Première connexion"
+Change = "Change"
+KleiCommunity = "Communauté Klei"
+OpenForums = "Ouvrir le Forum"
+Newsletter = "Newsletter"
+EditSubscriptions = "Mettre à jour les abonnements"
+UnsubscribeFromEmails = "Se désinscrire des courriels"
+SignupForABeta = "S'enregistrer pour la bêta"
+GetDRMFreeGames = "Obtenir un jeu sans DRM"
+DownloadPersonalData = "Télécharger ses informations personnelles"
+SeeMyGames = "Voir mes jeux"
+GamePlayerID = "ID Joueur"
+UserNotFound = "Joueur non trouvé"
+
+; Linked Accounts
+; https://accounts.klei.com/account/linked/list
+
+LinkedAccounts = "Comptes liés"
+LinkAccountButton = "Lier le compte"
+RemoveAccount = "Supprimer le compte"
+AddedOn = "Ajouté le"
+AccountLinkRulesTitle = "Règles de Streaming"
+AccountLinkRulesDescription = "Les récompenses Klei sont prévues pour aider les streamers qui créent du contenu de qualité pour notre communauté. Les streams AFK sont négatifs pour le système. Les streams AFK ou montrant du contenu autre que Don’t Starve ou Don’t Starve Together ne sont pas autorisés. Les streamers qui ne suivent pas ces règles seront dissociés et potentiellement interdits de participation ultérieure."
+AccountLinkRulesAccept = "Accepter et continuer"
+
+; Privacy Change Email
+; https://accounts.klei.com/account/email/change
+
+ChangeEmail = "Changer mon courriel"
+ChangeEmailDescription = "C'est l'adresse courriel qui nous utiliserons pour vous envoyer des nouveautés concernant votre compte ou vos jeux."

```

Comment:
`que nous utiliserons` =)
`envoyer les nouveautés à propos`

We can also turn it to the passive way, as we tend to use it way more in French that in english:
`Addresse de courriel où vous recevrez les nouveautés à propos de votre compte et de nos jeux.`